File: praktikum/C++/nilai_terkecil.c
#include <stdio.h>

int main() {
    int kecil=0;
    int angka=0;
    int i;
    
    printf("masukkan 10 angka: ");
    for(i=0; i<10; i++){
        scanf("%d", &angka);
        if(i==0){
            kecil=angka;
        }else if(angka<kecil){
            kecil=angka;
        }
    }

    printf("angka terkecil adalah %d", kecil);
    return 0;
}

File: praktikum/PYTHON/nilai_terkecil.py
kecil = 0
angka = 0
i = 0

print("masukkan 10 angka: ")
for i in range(10):
    angka=int(input())
    if i==0:
        kecil=angka
    elif angka < kecil:
        kecil= angka

print(f"angka terkecil adalah {kecil}")
